python/hail/context.py

```diff
@@ -465,6 +471,74 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npart
         return VariantDataset(self, jvds)
 
     @handle_py4j
+    def import_vcf_generic(self, path, force=False, force_bgz=False, header_file=None, npartitions=None,
```

Comment:
I like `import_vcf_generic` better. I prefer for the names to be identical with auto-complete and it's easier to remember the exact method name when there aren't too many differences between the two. @danking what do you think?